docs/src/Sheet.doc.js

```diff
@@ -23,472 +30,445 @@ card(
-    - Backdrop fade out
-    - Sheet slide out to the side
-
-    Once the **out** animation is finished, only then the \`onDismiss\` will be called to effectively to allow the parent component to unmount the \`<Sheet>\`.
-
-    ### 2) Animation on children and footer
-    The **in** animation is called with the initial render process from the entrypoint, with no option for customization.
-    The **out** animation, however, is called from the default exitpoints:
-      - ESC key
-      - Click on outside
-      - X button (header)
-
-    However, in order to provide an exitpoint on the elements passed to the \`children\` and \`footer\` props, the following render prop can be used:
-    \`\`\`
-    ({ onDismissStart }) => ( ... )
-    \`\`\`
-
-    When using this render prop, just pass the argument \`onDismissStart\` to your exitpoint action elements.
-
-    In the example below, please notice all the following animations:
-    - **in** (on show): from the "Open sheet" button entrypoint.
-    - **out** (on dismiss): from these 8 exitpoints:
-      - ESC key
-      - Click on outside
-      - X button (heading)
-      - Close button (subHeading)
-      - Right arrow icon red button (children)
-      - Done red button (children)
-      - Left arrow red icon button (children)
-      - Close button (footer)
-  `}
-    defaultCode={`
-function AnimationExample() {
-  const [shouldShow, setShouldShow] = React.useState(false);
-  const HEADER_ZINDEX = new FixedZIndex(10);
-  const sheetZIndex = new CompositeZIndex([HEADER_ZINDEX]);
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="Labels"
+      description={`
+- \`accessibilityDismissButtonLabel\`: provides a short, descriptive label for screen readers as a text alternative to the Dismiss button. Populates the \`aria-label\` attribute on the Dismiss button.
+- \`accessibilitySheetLabel\`: provides a short, descriptive label for screen readers to contextualize the purpose of Sheet. Please don’t repeat the same text being passed in the heading prop, but instead provide something that summarizes the Sheet’s purpose. For instance, if the \`heading\` is "Pin Builder", the \`accessibilitySheetLabel\` can be "Create a new Pin". Populates the \`aria-label\` attribute on the entire dialog.
+`}
+    >
+      <MainSection.Card
+        cardSize="lg"
+        iframeContent={`
+        <iframe src="https://codesandbox.io/embed/magical-christian-2kk1c?fontsize=14&hidenavigation=1&theme=light&view=preview"
+        style={{width: '100%', height:'600px', border:'0', borderRadius: '4px', overflow:'hidden'}}
+        title="Accessibility Labels"
+        allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+        sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+      ></iframe>
+        `}
+        defaultCode={`
```

Comment:
Ah, the one downfall of the sandbox approach: it's no longer live-editable within the docs page. Bummer. I don't think that's a blocker by any means, but it is something we should probably communicate to the user. Otherwise I expect confusion and frustration.